doc/util/literate_chapel.py

```diff
@@ -132,7 +146,10 @@ def push_line(line):
     flush()
     if chunk_index != 0:
         # We found __BREAK__; print the last chunk.
-        output.append(('output', chunk_index))
+        # output.append(('output', chunk_index))
+
+        # Don't print the last chunk, it's usually garbage anyway.
```

Comment:
Not sure I'm following... The last chunk is usually garbage because it just contains the newline at the end of the file?

Reply:
Yep: I've run into it many times in cases like the following.
Usually it contains no output at all. For instance,
```Chapel
// Here's some output!
writeln("Output");
// And that's the output. Time to conclude!
// The last output chunk will be printed below, but it's just empty.
```

I am fine with leaving out the last chunk; if the author wants output at the end, another `__BREAK__` will do that!

Reply:
Ok that makes sense, I just wanted to make sure I was tracking.